client/my-sites/plan-features/index.jsx

```diff
@@ -412,10 +418,26 @@ class PlanFeatures extends Component {
 				features,
 				planName
 			} = properties;
+			
+			let planFeatures = features;
```

Comment:
No apologies needed! I will refactor accordingly with the new variable naming, which makes total sense!